src/main.js

```diff
@@ -227,12 +228,30 @@ app.on('ready', function() {
 
     trayIcon.setToolTip(app.getName());
     trayIcon.on('click', function() {
+      if (process.platform === 'win32') {
+        if (config.minimizeToTray) {
+          if (mainWindow.isHidden) {
```

Comment:
Ok, I see your point. What if I change the close to happen on a double click? Another possibility would be a "Minimize to tray" menu entry in the context menu of the tray icon, although this would not as much as convenient. A third possibility would be, to enable this behaviour explicitely via the settings, so everyone can decide by itself, if this feature should be used or not. Whats your opinion here?